setup/lang/ged2gwb.htm

```diff
@@ -203,8 +212,31 @@ <h1 style="margin:0;">
 </table></dl>
 
 <dl><dt><dd><table style="border:1px solid" width="100%%">
-<tr align=left><td>&nbsp;</td>
+<tr align=left><td>&nbsp;&nbsp;&nbsp;</td>
 <td>[
+en: "first names enclosed": the -fne option must be followed by a 
+    two characters string "be". When creating a person, if the GEDCOM first name part holds
+    a part between 'b' (any character) and 'e' (any character), it
+    is considered to be the usual first name: e.g. -fne "\\" or -fne "()".
+fr: "first names enclosed" : l'option -fne doit &ecirc;tre suivie par une cha&icirc;ne de deux caract&egrave;ress "be".
+    A la cr&eacute;ation d'une personne, si une partie de la chaine  pr&eacute;nom commence par le caract&egrave;re 'b'
+    et se termine par le caract&egrave;re 'e', alors cette partie sera consid&eacute;r&eacute;e comme le pr&eacute;nom usuel.
+    Exemples :  e.g. -fne "\\" or -fne "()".
```

Comment:
Example taken from the code itself. Shows that the b and e characters can be the same.